### PR TITLE
Adding FAQ German

### PR DESCRIPTION
**Pull request by swat69**

### CODE REVIEW

File: _data/langs/de.yml [hunk 1]
```diff
@@ -197,3 +197,115 @@ footer:
...
+      premine:
+        question: "Wurde zum Start dieses Coins ein sog. Premine durchgeführt?"
+        answer: "Nein, Verge wurde ohne Premine erzeugt. Das bedeutet, dass alle Verge-Entwickler ihr Coins, wie alle anderen Personen auch, selber erworben haben und sich nicht durch ein Premine einen Vorteil verschafft haben."
+        answer2: "Falls du die Bedeutung von Premine nicht kennst: Premine bedeutet, dass sich die Entwickler eine größere Anzahl von Coins mit Hilfe einer speziellen Adresse für sich selbst reservieren und erst dann die restlichen Coins für die Allgemeinheit freigeben"
+      totalcoinsupply:
+        question: "Wieviel Vergecoins gibt es ingesamt?"
+        answer: "Es gibt insgesamt <b>~16.5 Milliarden</b>" Coins (exakt sind es 16,555,000,000). Wichtig dabei ist, dass bisher nicht alle Coins erzeugt wurde, dies erfolgt zeitanhängig. ~9 Milliarden Coins wurden im ersten Jahr (2014) erzeugt, seitdem wurden bzw. werden jedes Jahr weitere 1 Milliarde Coins erzeugt. Dies setzt sich so lange fort bis alle Coins verteilt wurden."
```
Comment (gfranko) [inline _data/langs/de.yml]: Looks like there is an extra double quotes added.... should be:

```javascript
answer: "Es gibt insgesamt <b>~16.5 Milliarden</b> Coins (exakt sind es 16,555,000,000). Wichtig dabei ist, dass bisher nicht alle Coins erzeugt wurde, dies erfolgt zeitanhängig. ~9 Milliarden Coins wurden im ersten Jahr (2014) erzeugt, seitdem wurden bzw. werden jedes Jahr weitere 1 Milliarde Coins erzeugt. Dies setzt sich so lange fort bis alle Coins verteilt wurden."
```

File: _data/langs/de.yml [hunk 2]
```diff
@@ -197,3 +197,115 @@ footer:
...
+      richlist:
+        question: "Wo kann ich eine Liste der Personen mit den meisten Coins (RichList) einsehen"
+        answer: "Die Übersicht steht <a href='http://vergecurrency.info/richlist' target='_blank'>hier</a> zu Verfügung"
+      wheretolearnmore:
+        question: "Was kann ich machen, um mehr über Verge zu erfahren?"
+        answer: "Wird sind in vielen <a href='https://vergecurrency.com/#social' target='_blank'>sozialen</a> Netzwerken vertreten und besonders aktiv auf Twitter, Telegram, Slack und Discourse."
+      roadmap:
+        question: "Wo finde ich die Verge Roadmap?"
+        answer: "Die <a href='https://vergecurrency.com/#roadmap' target='_blank'>Roadmap</a>" steht auf unserer Website zur Verfügung und zeigt normalerweise die Entwicklung der nächsten ein oder zwei Monate. Wir wollen nicht, dass Punkte mit Realisierungstermin weit in der Zukunft (weiter als 3 Monate) aufgeführt werden, die Dinge ändern sich einfach zu schnell in der Crypt-Welt."
```
Comment (gfranko) [inline _data/langs/de.yml]: Looks like there is also another double quotes added to this string... should be:

```javascript
answer: "Die <a href='https://vergecurrency.com/#roadmap' target='_blank'>Roadmap</a> steht auf unserer Website zur Verfügung und zeigt normalerweise die Entwicklung der nächsten ein oder zwei Monate. Wir wollen nicht, dass Punkte mit Realisierungstermin weit in der Zukunft (weiter als 3 Monate) aufgeführt werden, die Dinge ändern sich einfach zu schnell in der Crypt-Welt."
```

File: _data/langs/de.yml (hunk 1 above)
Comment (swat69) [inline _data/langs/de.yml]: yes your are right

File: _data/langs/de.yml (hunk 2 above)
Comment (swat69) [inline _data/langs/de.yml]: same ok.

**Comment by swat69** [comment]
looks good

**Comment by gfranko** [comment]
💯  thanks!

**Pull request merged**